### PR TITLE
Add AdV INSPIRE SLDs

### PR DESCRIPTION
**Pull request by Claudia-WE**
Add AdV INSPIRE SLDs
For further clarification see: https://wetransform.atlassian.net/jira/servicedesk/projects/WGS/queues/custom/3/WGS-1356

### CODE REVIEW

File: inspire-slds-master/inspire-slds-master/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# ignore files from eclipse editor
```
Comment (JohannaOtt) [inline inspire-slds-master/inspire-slds-master/.gitignore:1]: The gitignore file should not be pushed to the repository IMO.
It seems like they extracted the source folder from a github repository themselves and zipped them without removing the gitignore file but now, they only want to provide the files publicly.

File: inspire-slds-master/inspire-slds-master/Vgl_SLD_AdV_ELF/ELF_SLDs.url [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[{000214A0-0000-0000-C000-000000000046}]
```
Comment (JohannaOtt) [inline inspire-slds-master/inspire-slds-master/Vgl_SLD_AdV_ELF/ELF_SLDs.url:1]: I wonder if the "Vgl_SLD_AdV_ELF" folder should be made public as well. Might make sense to check back if the whole content of the zip file sent should be contained.

**Review by JohannaOtt — CHANGES_REQUESTED** [review]
I added some comments to the changed files.
Additionally:
* As the WGS ticket is visible to AdV members anyway, I would have mentioned it in the commit message.
* The folder name "inspire-slds-master/inspire-slds-master" should be changed to "inspire-slds" or similar IMO

File: inspire-slds-master/inspire-slds-master/Vgl_SLD_AdV_ELF/ELF_SLDs.url (hunk 2 above)
Comment (Claudia-WE) [reply inspire-slds-master/inspire-slds-master/Vgl_SLD_AdV_ELF/ELF_SLDs.url]: I would keep "Vgl_SLD_AdV_ELF" as I don't see any harm to keep it and it was provided in the ZIP

**Comment by JohannaOtt** [comment]
@Claudia-WE Is it correct that these changes are covered in [this PR?](https://github.com/wetransform/adv-inspire-alignments/pull/68) Then this one should be closed.

**Comment by Claudia-WE** [comment]
> Is it correct that these changes are covered in https://github.com/wetransform/adv-inspire-alignments/pull/68 Then this one should be closed.

Yes

**Pull request closed without merging**